$OpenBSD: patch-src_PlatformSpecific_h,v 1.1 2012/10/25 07:59:45 dcoppa Exp $

Like FAT, NTFS doesn't like special characters either
(upstream git commit 41113ecf15019301afea6cb35d9c35a7b8a0bdd1)

--- src/PlatformSpecific.h.orig	Thu Oct 25 11:09:56 2012
+++ src/PlatformSpecific.h	Thu Oct 25 11:10:54 2012
@@ -100,6 +100,7 @@ inline bool CanFSHandleSpecialChars(const CPath& path)
 {
 	switch (GetFilesystemType(path)) {
 		case fsFAT:
+		case fsNTFS:
 		case fsHFS:
 			return false;
 		default:
